### ISSUE TITLE
WebIRC Support

### ISSUE THREAD

**Issue by Havvy** [Havvy/irc-socket#16]
See https://github.com/ircanywhere/irc-factory/issues/27

**Comment by hohl** [comment]
I've just had a look at the irc-socket.js source code. Pretty clean and short pice of code. For adding WebIRC support I guess it's just about a line like

    socket.raw(['WEBIRC', ...]);

before [line 117 in irc-socket.js](https://github.com/Havvy/irc-socket/blob/master/irc-socket.js#L117).

**Comment by Havvy** [comment]
I should apologize for letting this sit around. I was trying something largely different in handling errors and CAP (which involves irc-socket@3.0.0) and I found a problem that dooms my entire approach should you want to have IRCv3 tags (e.g. the timestamp tag) and wasn't sure what to do. But here's my plan:

1) I merged in your PR and published a new version of irc-socket.  v.2.4.0.
2) I'm going to publish v.3.0.0 even with the caveat that some IRCv3 tags won't work.
3) I may at some point move the initialization logic added in v.3.0.0 to something on top of irc-message which does parse the messages.

It's your choice whether you use v2.4.0 or v.3.0.0. If you don't anything else, I'd stick with v.2.4.0 personally.
